dlrover/python/master/monitor/error_monitor.py

```diff
@@ -151,5 +151,5 @@ def _handle_node_error(self, node: Node, error_data: str):
         if self.cordon_node_eanbled:
             succeed = self._k8s_client.cordon_node(node.host_name)
             if succeed:
-                logger.info(f"Node {node.name} is marked unschedulable.")
+                logger.info(f"Host {node.host_name} is marked unschedulable.")
```

Comment:
'Node' is a domain model in DLRover. Also, the 'host_name' can be a long string(including idc and zone info), is not readable for user.

Reply:
Yes, Node is a domain model, however, for K8s platform, the domain model is implemented by K8s Pod. Since the POD is ephemeral, we can not mark a pod as unschedulable, we can only mark the host of a pod as unschedulable. 
So, the origianl log here might be confusing.

Reply:
U r right, the target model here is the K8s Node.